Appendix.Rmd

```diff
@@ -55,7 +80,7 @@ It's always a good idea to refer to the the primary literature! The following ar
 - @cosgriff_citation_2020 - The promise of shared data for addressing public health emergencies, such as the SARS-CoV-2 pandemic
 - @mckiernan_citation_2016 - Data sharing can help advance careers through increases in citations, media attention, potential collaborators, job opportunities, and funding opportunities
 - @piwowar_citation_2011 - Investing in data storage and archiving is a good investment of grant funds
-- @sansone_fairsharing_2019 - FAIRsharing as a community approach to standards, repositories and policies
+- @sansone_citation_2019 - FAIRsharing as a community approach to standards, repositories and policies
```

Comment:
oh did I not add my book.bib updates for this last time?

Reply:
I didn't see it, it happens :)